CONTRIBUTING.md

```diff
@@ -67,7 +67,7 @@ This is because one may not be emulating bash environment in any way or doing it
 #### macOS
```

Comment:
u added an empty line after `####` section below but here not. be consistent

Reply:
Ha! nope, it's just that at the end of each `##` group there are two empty lines